cmd/kubeapps-apis/docs/kubeapps-apis.swagger.json

```diff
@@ -1826,6 +1901,58 @@
       "description": "Response for GetInstalledPackageSummaries",
       "title": "GetInstalledPackageSummariesResponse"
     },
+    "v1alpha1InstalledPackageDetail": {
+      "type": "object",
+      "properties": {
+        "installedPackageRef": {
+          "$ref": "#/definitions/v1alpha1InstalledPackageReference",
+          "description": "A reference uniquely identifying the package.",
```

Comment:
should we say : "identifying the installed package" instead of "identifying the package"?